framework/include/linearfvkernels/LinearFVTimeDerivative.h

```diff
@@ -0,0 +1,57 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "LinearFVElementalKernel.h"
+#include "TimeIntegrator.h"
+
+/**
+ * Kernel that adds contributions from a time derivative term to a linear system
+ * populated using the finite volume method.
+ */
+class LinearFVTimeDerivative : public LinearFVElementalKernel
+{
+public:
+  static InputParameters validParams();
+
+  /**
+   * Class constructor.
+   * @param params The InputParameters for the kernel.
+   */
+  LinearFVTimeDerivative(const InputParameters & params);
+
+  virtual Real computeMatrixContribution() override;
+
+  virtual Real computeRightHandSideContribution() override;
+
+  /**
+   * Set the current ElemInfo object.
+   * @param elem_info The new ElemInfo which will be used as the current
+   */
+  virtual void setCurrentElemInfo(const ElemInfo * elem_info) override;
+
+protected:
+  /// The functor for the material property multipler
+  const Moose::Functor<Real> & _factor;
+
+  /// The time integrator to use in this kernel, will provide information
+  /// on how many states are required in the history.
+  const TimeIntegrator & _time_integrator;
+
+private:
+  /// Current values of the material property multiplier. Used
```

Comment:
Current values where?

Reply:
The rest of the doc string and then the doc string for the next data member make it seem as if this is a time history, in which case `Current` doesn't fit